HtmlDialogDemos/HtmlDialogDemosTest/src_test/ch/ivyteam/htmldialog/server/test/EngineUrl.java

```diff
@@ -29,9 +32,14 @@ public static String applicationName()
   {
     return System.getProperty("test.engine.app", DESIGNER);
   }
-
-  public static Boolean isDesigner()
+  
+  public static void startProcess(String pathToIvp)
   {
-    return Boolean.valueOf(EngineUrl.applicationName() == DESIGNER);
+    Selenide.open(EngineUrl.process() + "/HtmlDialogDemos/" + pathToIvp);
+    if (StringUtils.containsNone(pathToIvp, "ClientSideValidationDemo"))

```

Comment:
!? shouldn't this live somewhere else?

Reply:
Was in the webbase class before... and this don't longer exists.